app/models/study_json_record/processor_v2.rb

```diff
@@ -227,6 +227,13 @@ def id_information_data
   end
 
   def ipd_information_types_data
+    return unless protocol_section
+
+    nct_id = protocol_section.dig('identificationModule', 'nctId')
+    ipd_sharing_info_types = protocol_section.dig('ipdSharingStatementModule', 'infoTypes')
+    return unless ipd_sharing_info_types
+
+    { nct_id: nct_id, name: ipd_sharing_info_types }

```

Comment:
We need to return an array with the elements format `{nct_id: "NCTXXX", name: "SAP"}`

Reply:
Fixed `ipd_information_types_data` to return array instead of hash.